### PR TITLE
Use dart analyze for CI

### LINKED ISSUE #223: Public members lack documentation
Packages with public APIs are more appropriate to analyze using `dart` rather than `flutter`
However, it is currently reported that there is lack documentation for 59 public members.
after solving all of this, we have to use `dart` instead of `flutter`

- [x] messageport/example/lib/main.dart:6:14 • Document all public members. • public_member_api_docs
- [x] messageport/example/lib/main.dart:7:14 • Document all public members. • public_member_api_docs
- [x] messageport/example/lib/main.dart:29:7 • Document all public members. • public_member_api_docs
- [x] messageport/lib/messageport_tizen.dart:84:16 • Document all public members. • public_member_api_docs
- [x] messageport/lib/src/messageport_manager.dart:12:7 • Document all public members. • public_member_api_docs
- [x] messageport/lib/src/messageport_manager.dart:13:3 • Document all public members. • public_member_api_docs
- [x] messageport/lib/src/messageport_manager.dart:15:16 • Document all public members. • public_member_api_docs
- [x] messageport/lib/src/messageport_manager.dart:22:16 • Document all public members. • public_member_api_docs
- [x] messageport/lib/src/messageport_manager.dart:33:16 • Document all public members. • public_member_api_docs
- [x] messageport/lib/src/messageport_manager.dart:43:16 • Document all public members. • public_member_api_docs
- [x] messageport/lib/src/messageport_manager.dart:56:19 • Document all public members. • public_member_api_docs
- [x] package_info_plus/lib/package_info_plus_tizen.dart:18:15 • Document all public members. • public_member_api_docs
- [x] share_plus/lib/share_plus_tizen.dart:18:15 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/app_control.dart:20:3 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:9:7 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:11:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:14:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:15:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:17:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:18:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:20:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:21:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:24:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:25:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:26:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:27:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:29:1 • Document all public members. • public_member_api_docs
- [x] url_launcher/lib/src/types.dart:30:1 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/example/lib/custom_page_view.dart:6:7 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/example/lib/custom_page_view.dart:7:3 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/example/lib/custom_page_view.dart:9:14 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/example/lib/main.dart:12:7 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/lib/wearable_rotary.dart:7:25 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/lib/wearable_rotary.dart:16:6 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/lib/wearable_rotary.dart:17:3 • Document all public members. • public_member_api_docs
- [x] wearable_rotary/lib/wearable_rotary.dart:18:3 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:15:7 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:16:3 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:30:16 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:31:38 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:32:37 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:33:24 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:34:53 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:35:17 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:36:32 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:171:7 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:207:14 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:213:12 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:217:16 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:268:16 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:282:11 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:284:7 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:288:12 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:290:8 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:303:16 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:376:7 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:400:30 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/src/platform_view_tizen.dart:453:27 • Document all public members. • public_member_api_docs
- [x] webview_flutter/lib/webview_flutter_tizen.dart:28:15 • Document all public members. • public_member_api_docs

_Originally posted by @bbrto21 in https://github.com/flutter-tizen/plugins/pull/218#discussion_r708765550_

### PR DESCRIPTION
**Pull request by HakkyuKim**
Closes #223.

### CODE REVIEW

**Comment by swift-kim** [comment]
Here's my suggestion: https://github.com/swift-kim/plugins/commit/b37d29f5665a5616df48afe78212f7617a70c9c7.

File: .github/workflows/analyze.yml [hunk 1]
```diff
@@ -9,19 +9,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v1
         with:
-          flutter-version: "2.5.1"
+          channel: stable
```
Comment (bwikbs) [inline .github/workflows/analyze.yml:12]: I think it's right to change it like this, but I'm afraid that often unintentional failures will occur.

Comment (HakkyuKim) [reply .github/workflows/analyze.yml:12]: Maybe on new flutter releases?

Comment (swift-kim) [reply .github/workflows/analyze.yml:12]: Actually I and @bbrto21 have discussed about this matter before. @bbrto21 Do you remember the conclusion?

Comment (bbrto21) [reply .github/workflows/analyze.yml:12]: I agreed to use the stable channel, but it didn't works properly on this repository at that time.(I don't remember exactly why it didn't work)
If there are no more problems, I prefer to use the stable channel.

Comment (swift-kim) [reply .github/workflows/analyze.yml:12]: I can't think of any unwanted problem (false alarm) that might be caused by this change.

- A dependency resolution failure (like [this](https://github.com/flutter-tizen/plugins/pull/125#issuecomment-866661681)) may occur if test packages (such as flutter_test) have updated dependencies on new Flutter releases.
- Analysis or formatting rules are added/changed by new versions of Flutter/Dart.

In both cases, the errors should not be suppressed and should be addressed as quick as possible.

Comment (HakkyuKim) [reply .github/workflows/analyze.yml:12]: Okay, I see that CI will detect problems early which will allow us to solve them as soon as possible.

**Review by bwikbs — APPROVED** [review]
Thanks!!! :+1: :+1: :+1:

**Review by swift-kim — APPROVED** [review]

**Pull request merged**